src/Spago/Packages.hs

```diff
@@ -349,7 +349,18 @@ data CheckModulesUnique = DoCheckModulesUnique | NoCheckModulesUnique
 verify :: Maybe CacheFlag -> CheckModulesUnique -> Maybe PackageName -> Spago ()
 verify cacheFlag chkModsUniq maybePackage = do
   logDebug "Running `spago verify`"
-  Config{ packageSet = packageSet@PackageSet{..}, ..} <- Config.ensureConfig
+
+  packageSet@PackageSet{..} <- do
+    -- Try to read a "packages.dhall" directly
+    try (liftIO (Dhall.inputExpr $ "./" <> PackageSet.packagesPath)) >>= \case 
```

Comment:
I'd invert the order here: we should try to read a config first (as that is present most often than the `packages.dhall`), and only if that fails we try to read the `packages.dhall` file

Reply:
@f-f If I put `ensureConfig` first then we'll still see its `die` error message even if the verify command later succeeds because it finds a `package.json` :disappointed: 

I can leave it as-is, or we could leave this until I/we sort out the `die` situation?

Reply:
Right, let's add a `FIXME` or `TODO` here and leave it as is for now